css/modern.css

```diff
@@ -901,6 +901,12 @@ span#updates-count {
     background-color: var(--active) !important;
 }
 
+[page^="/course/"] .sExtlink-processed:hover > .inline-icon.mini, [page^="/course/"] .inline-icon.mini:hover {
+    background: url(/sites/all/themes/schoology_theme/images/icons_sprite_new.png?5f5c25b861784536), var(--muted-text);
+    border: 2px solid var(--muted-text);
+    background-position: 0 -1450px;
+}
+
```

Comment:
I'm going to modify this to avoid using the page selector. In general I don't like using it since it's kind of an ugly hack.

Reply:
Yeah ok, was just trying to ensure that it doesn't break anything else.